app/forms/concerns/hyku_addons/work_form.rb

```diff
@@ -47,15 +47,15 @@ def available_terms
       def hyrax_terms
         %i[visibility files visibility_during_embargo embargo_release_date visibility_after_embargo
            visibility_during_lease lease_expiration_date visibility_after_lease admin_set_id member_of_collection_ids
-           ordered_member_ids in_works_ids rendering_ids]
+           ordered_member_ids in_works_ids rendering_ids representative_id thumbnail_id]
       end
 
       def common_fields
```

Comment:
Do we also need to add `rendering_ids` to the `common_fields`?

Reply:
Looks like, yes. I've made the test of removing it and the deposit form wouldn't allow me to edit the _Rendering_ select box.